### ISSUE TITLE
Assign window tabs (Text To Image, etc) to hotkeys (⌘-1, 2, etc)

### ISSUE THREAD

**Issue by Leland** [divamgupta/diffusionbee-stable-diffusion-ui#178]
Switching between tabs should be exposed as a hotkey for keyboard users.

I recommend adding the Window menu to the macOS menubar, adding each of the tabs to it (Text To Image, Image To Image, Outpainting, History), and mapping each one to a number. E.g. <kbd>⌘</kbd>+<kbd>1</kbd> for Text To Image.

**Comment by Leland** [comment]
I tried to get this to work, despite my lack of Vue and Electron experience. The menu [lives here](https://github.com/divamgupta/diffusionbee-stable-diffusion-ui/blob/master/electron_app/src/menu_template.js). I think the hotkeys need to fire an ipcRenderer event, which then gets caught by the application?
